theia-flutter-docker/latest.package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia Flutter Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/editor-preview": "latest",
+        "@theia/file-search": "latest",
+        "@theia/getting-started": "latest",
+        "@theia/git": "latest",
+        "@theia/json": "latest",
+        "@theia/markers": "latest",
+        "@theia/messages": "latest",
+        "@theia/navigator": "latest",
+        "@theia/outline-view": "latest",
+        "@theia/plugin-ext-vscode": "latest",
+        "@theia/preferences": "latest",
+        "@theia/preview": "latest",
+        "@theia/search-in-workspace": "latest",
+        "@theia/terminal": "latest",
+        "@theia/textmate-grammars": "latest"
+    },
+    "devDependencies": {
+        "@theia/cli": "latest",
+        "@theia/debug": "latest"
+    },
+    "resolutions": {
+        "vscode-languageserver-protocol": "3.15.0-next.9",
+        "vscode-languageserver-types": "3.15.0-next.5",
+        "**/vscode-json-languageserver/**/vscode-languageserver": "6.0.0-next.1"
+    },
+    "scripts": {
+        "postinstall": "download-debug-adapters"
+    },
+    "adapterDir": "plugins",
+    "adapters": {
+        "dart-code": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/Dart-Code/extension/dart-code/3.2.1/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage",
+        "flutter": "https://ms-vscode.gallery.vsassets.io/_apis/public/gallery/publisher/Dart-Code/extension/flutter/3.8.1/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage"
```

Comment:
Same for this extension. However I do not see any `.vsix` in their [repo's releases](https://github.com/Dart-Code/Flutter/releases)

Reply:
@marcdumais-work I opened an issue on their repository asking if they'd be open to also including the `.vsix` as part of the releases: https://github.com/Dart-Code/Dart-Code/issues/2254

Do you remember the exact reason why we could not download directly from the marketplace? https://github.com/Dart-Code/Dart-Code/issues/2254#issuecomment-585072390

Reply:
> @marcdumais-work I opened an issue on their repository asking if they'd be open to also including the `.vsix` as part of the releases: [Dart-Code/Dart-Code#2254](https://github.com/Dart-Code/Dart-Code/issues/2254)

Thanks Vince. 

> Do you remember the exact reason why we could not download directly from the marketplace? [Dart-Code/Dart-Code#2254 (comment)](https://github.com/Dart-Code/Dart-Code/issues/2254#issuecomment-585072390)

Yes - it's prohibited by the Marketplace terms of service:

Start at the main [marketplace page](https://marketplace.visualstudio.com), scroll down to the bottom and click on "terms of use" on the bottom right side.

From section 1b:
"Marketplace Offerings are intended for use only with Visual Studio Products and Services and you may only install and use Marketplace Offerings with Visual Studio Products and Services. "

Reply:
> > @marcdumais-work I opened an issue on their repository asking if they'd be open to also including the `.vsix` as part of the releases: [Dart-Code/Dart-Code#2254](https://github.com/Dart-Code/Dart-Code/issues/2254)
> 
> Thanks Vince.
> 
> > Do you remember the exact reason why we could not download directly from the marketplace? [Dart-Code/Dart-Code#2254 (comment)](https://github.com/Dart-Code/Dart-Code/issues/2254#issuecomment-585072390)
> 
> Yes - it's prohibited by the Marketplace terms of service:
> 
> Start at the main [marketplace page](https://marketplace.visualstudio.com), scroll down to the bottom and click on "terms of use" on the bottom right side.
> 
> From section 1b:
> "Marketplace Offerings are intended for use only with Visual Studio Products and Services and you may only install and use Marketplace Offerings with Visual Studio Products and Services. "

Thank you! I wanted to give a proper answer when responding to the dart-code issue.

Reply:
@marcdumais-work they were so kind as to include the `.vsix` in their releases for both **flutter** and **dart** extensions: https://github.com/Dart-Code/Dart-Code/issues/2254#issuecomment-585302750

Reply:
Awesome - good idea to touch-base with them @vince-fugnitto.